pkg/endpoint/policy.go

```diff
@@ -257,61 +155,54 @@ func (e *Endpoint) resolveL4Policy(owner Owner, repo *policy.Repository, c *poli
 	}
 
 	newL4EgressPolicy, err := repo.ResolveL4EgressPolicy(&egressCtx)
+
+	log.Errorf("ENDPOINT %d newL4EgressPolicy: %s", e.ID, newL4EgressPolicy)
```

Comment:
I assume this is supposed to be `log.Debugf()`?

Reply:
Whoops, thought I had deleted this. Will remove in future upload!